### PR TITLE
Corrects gort arduino upload command

### PR DESCRIPTION
**Pull request by lcguida**
`gort` command should be `gort arduino upload firmdata <port>` instead of `gort arduino upload <port>`, as stated here http://artoo.io/documentation/getting-started/tutorials/arduino/ and here https://github.com/hybridgroup/gort/issues/38. Following this tutorial leads the user to an error.

### CODE REVIEW

File: source/documentation/platforms/arduino.html.haml [hunk 1]
```diff
@@ -88,7 +88,7 @@ subnavjs: true
     Once the `avrdude` uploader is installed, we can upload the Firmata protocol to the Arduino.
     Use the serialport address we found with the earlier `gort`, or leave it blank to use the default address (`/dev/ttyACM0`).
 
-        $ gort arduino upload /dev/ttyACM0
+        $ gort arduino upload firmdata /dev/ttyACM0
```
Comment (deadprogram) [inline source/documentation/platforms/arduino.html.haml]: Thanks for the fix. However, I think this should be:

```
$ gort arduino upload firmata /dev/ttyACM0
```

Comment (lcguida) [inline source/documentation/platforms/arduino.html.haml]: Done.

**Comment by deadprogram** [comment]
Thank you, @lcguida

**Comment by lcguida** [comment]
Glad to help. Great work by the way. Real fun work on robotics with ruby.

**Pull request merged**